compaction/compaction_manager.cc

```diff
@@ -1034,8 +1034,14 @@ void compaction_manager::register_metrics() {
 }
 
 void compaction_manager::enable() {
-    SCYLLA_ASSERT(_state == state::none || _state == state::disabled);
-    _state = state::enabled;
+    SCYLLA_ASSERT(_state == state::none || _state == state::running);
+
+    if (_disabled_state_count > 0 && --_disabled_state_count > 0) {
+        cmlog.warn("Compaction manager is still disabled, requires {} more call(s) to enable()", _disabled_state_count);
```

Comment:
nit: not sure that a warning is needed if this can happen  under normal circumstances

Reply:
Should have been debug as for the remaining services.